src/couch_views/src/couch_views_indexer.erl

```diff
@@ -407,17 +408,26 @@ fetch_docs(Db, Changes) ->
         }
     end, #{}, erlfdb:wait_for_all(RevFutures)),
 
+    AddLocalSeq = lists:keymember(<<"local_seq">>, 1, DesignOpts),
```

Comment:
`keymember` will indicate that the option is present only? Maybe we want a keyfind here http://erlang.org/doc/man/lists.html#keyfind-3 then if tuple is present check the value?

Reply:
@nickva good spot. I've changed it to use `get_value`